dev/conductor/lib/codesign.dart

```diff
@@ -112,29 +112,33 @@ class CodesignCommand extends Command<void> {
           'the desired revision and run that version of the conductor.\n');
       revision = argResults![kRevision] as String;
     } else {
-      revision = (processManager.runSync(
+      revision = ((await processManager.run(
         <String>['git', 'rev-parse', 'HEAD'],
-        workingDirectory: framework.checkoutDirectory.path,
-      ).stdout as String).trim();
+        workingDirectory: (await framework.checkoutDirectory).path,
+      ))
+              .stdout as String)
```

Comment:
this is the reason our style guide forbids using `dartfmt`. this formatting change should be reverted.